astropy/wcs/tests/conftest.py

```diff
@@ -4,7 +4,7 @@
 
 from astropy import wcs
 
-from . helper import SimModelTAB
```

Comment:
Whoa, how did this not fail CI?

Reply:
It's imported but never used?

Reply:
never mind. that import is not removed. just the space.